sys-apps/hwinfo/hwinfo-21.23.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit multilib rpm toolchain-funcs
+
+DESCRIPTION="hardware detection tool used in SuSE Linux"
+HOMEPAGE="http://www.opensuse.org/"
+SRC_URI="http://download.opensuse.org/source/factory/repo/oss/suse/src/${P}-1.2.src.rpm"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~arm-linux ~x86-linux"
+IUSE=""
+
+RDEPEND="
+	amd64? ( dev-libs/libx86emu )
+	x86? ( dev-libs/libx86emu )"
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	>=sys-kernel/linux-headers-2.6.17"
+
+MAKEOPTS="${MAKEOPTS} -j1"
+
+src_unpack() {
+	rpm_src_unpack ${A}
+}
+
+src_prepare() {
+	# Respect AR variable.
+	sed -i \
+		-e 's:ar r:$(AR) r:' \
+		src/{,isdn,ids,smp,hd}/Makefile || die
+
+	# Avoid -I directories for dbus because HAL is obsolete.
+	sed -i -e '/CFLAGS/d' src/hd/Makefile || die
```

Comment:
@jakeogh when you are bumping ebuilds, you need to check if all the code is still necessary, not only that it builds. This sed is superfluous.